sdk/src/pubkey.rs

```diff
@@ -85,7 +85,7 @@ impl Pubkey {
     }
 
     pub fn create_program_address(
-        seeds: &[&str],
+        seeds: &[&[u8]],
```

Comment:
Do you still need `as_ref` in the implementation?

Reply:
yea